### ISSUE TITLE
Consider creating a bridge + MEW bundle for offline use

### ISSUE THREAD

**Issue by x1ddos** [digitalbitbox/bitbox-bridge#9]
This came up while discussing https://github.com/digitalbitbox/bitbox-bridge/issues/7.

Currently, users need to run a separate static file server like `python3 -m http.server --bind 127.0.0.1` and open `http://localhost:port` to access [MEW offline](https://kb.myetherwallet.com/en/offline/using-mew-offline/). Opening an HTML file directly with the browser is suboptimal. For example, see [security considerations](https://tools.ietf.org/html/rfc8089#section-5) and [corner cases](https://en.wikipedia.org/wiki/Same-origin_policy#Corner_cases).

It feels like a bundle of MEW + the bridge including a built-in static file server would be very helpful for those who would like to use BitBox02 with MEW offline.

Such a bundle would nicely package everything and avoid running a separate static file server manually.

**Comment by benma** [comment]
I don't know if something changed on MEW's side, but if I clone their repo and do 

```
npm i
npm run dev
```

I get a fully functioning MEW that works with the BitBox02, locally.

**Comment by x1ddos** [comment]
The idea here was to create a complete offline bundle but `npm i` would need internet connection.

**Comment by benma** [comment]
Does MEW even do anything when offline? It also needs to fetch balances etc. somehow. Or maybe it can be used to just sign offline, that would be nice.

**Comment by x1ddos** [comment]
Yeah, that's exactly what it seems to say on https://kb.myetherwallet.com/en/offline/using-mew-offline/

> The main computer being used to generate the transactions should not be connected to the internet. You can generate a wallet completely offline and send transactions on a different computer, which must be connected to the internet, from our ‘Send Offline’ section.
